portfolio/src/main/webapp/index.html

```diff
@@ -11,17 +11,16 @@
   <script src="script.js"></script>
 </head>
 
-<body>
+<body  onload="getCommentSection()">
   <div class:"container-fluid">
-
     <div>
       <div class="collapse" id="navbarToggleExternalContent">
         <div class="bg-dark p-4">
           <h5 class="text-white h4">Menu</h5><br>
-          <a href="index.html" class="btn btn-outline-light" role="dark">Home</a><br><br>
-          <a href="pictures.html" class="btn btn-outline-light" role="dark">Pictures</a><br><br>
-          <a href="hobbies.html" class="btn btn-outline-light" role="dark">Hobbies</a><br><br>
-          <a href="resume.html" class="btn btn-outline-light" role="dark">Resume</a><br>
+          <a href="index.html" class="btn btn-outline-light margin-bottom" role="dark">Home</a><br>
```

Comment:
(super) nit: It can be a little confusing to have class names that are the same as CSS property names. Consider something like "menu-links" instead.

Reply:
fixing in the subbranch